vnext/src/index.windows.js

```diff
@@ -10,6 +10,7 @@
 
 'use strict';
```

Comment:
Should this file be using, or at least have a shared file with index.windows.js? I see the same changes.

Reply:
Sharing between packages has come up before, but we haven't been able to find a pattern that makes it worth it. The changes here are all to an upstream `index.js` that both the NetUI and XAML platforms fork in different ways.